### PR TITLE
Total commission earned for each sales emp

### PR DESCRIPTION
**Pull request by OdhranHague**
PB 13. As a member of the HR team I want to see a list of sales employees and how much each has made in commission this year

### CODE REVIEW

**Review by Samuel-BlankAmber — APPROVED** [review]
Brill

File: sql/TotalCommision.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
+ALTER TABLE SalesEmployee RENAME COLUMN SaleID TO SalesEmpID;
+
+SELECT * FROM Project;
+SELECT * FROM SalesEmployee;
```
Comment (jkelso662) [inline sql/TotalCommision.sql]: Only be committing complete and finished code

File: sql/TotalCommision.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ADD column SalesEmpID INT;
+
+ALTER TABLE Project
+ADD FOREIGN KEY(SalesEmpID) REFERENCES SalesEmployee(SalesEmpID);
+
+-- As a member of the HR team I want to see a list of sales employees and 
+-- how much each has made in commission this year
+
+SELECT CONCAT(Forename,' ', Surname) as 'Name', ProjectValue, ComRate FROM SalesEmployee s
+INNER JOIN Project p on p.SalesEmpID = s.SalesEmpID;
```
Comment (jkelso662) [inline sql/TotalCommision.sql]: Only be committing complete and finished code

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Nearly! Only be committing complete and finished code. Also next time think about where alter table stuff should be. Otherwise not bad! :)

**Review by jkelso662 — APPROVED** [review]
Much better! I like it :)

**Pull request merged**